contexts/generic

```diff
@@ -18,11 +18,18 @@
                 "@id": "schema:image",
                 "@type": "@id"
```

Comment:
shouldn't this also be `xsd:anyURI` given how we are using it? or can people put an `ImageObject` there as well?

Reply:
given the shacl spec, let's remove `@type` from here. and put it in the examples. alternatively you can keep this and add another key `imageurl` and make that have `"@type": "xsd:anyURI"` in the graph both would map to schema:image, so the shacl would be fine. but in json, the author has to use image for ImageObject and imageurl for urls.